modules/ossm-observability-about.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/observability/ossm-observability-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-observability-about_{context}"]
+= About metrics, logs, and traces in Service Mesh
+:context: ossm-observability-about
+
+//Parts taken from existing "Observaility overview"
+//08/16/2024: Observability Attributes are not yet part of the _attributes file for service-mesh-docs-main. If they are not added for OSSM 3.0 TP1, they will be for OSSM 3.0 GA.
+Red Hat OpenShift Observability components, when used with {SMProductName}, provides real-time visibility, monitoring, and analysis of various system metrics, logs, traces, and events to help you quickly diagnose and troubleshoot issues before they impact systems or applications.
+
+You can add the following Observability components to your Mesh:
+
+{KialiProduct}:: Provides observability for your service mesh. You can view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
```

Comment:
I would say that Kiali helps you visualize your mesh. Not so much that it provides observability for your mesh. It does help you "observe" your mesh but it doesn't provide any of the observability "signals" itself (metrics, logs, traces). It simply visualizes those signals.